### DISCUSSION TITLE
Logic Pro Block Size minimum steps of 1024?

### DISCUSSION THREAD

**Discussion by simcc** [apohl79/audiogridder#1301] [Q&A]
Hi I'm testing this out with Logic 10.8.1 Ventura Mac Studio client...to windows 10 server. Latest Audiogridder on both. Direct connection 1G. 

The only block size I can set is 0, 1024, 2048 etc. samples. It's stuck on that no matter what I set Logic's buffer size to? Is that correct? I see people talking of using smaller blocks, is this just a Logic issue?

Also I get drop outs even at 1x 1024 block, so have to use 2x block (2048), meaning 43ms latency...even with just one plugin...I wonder am I doing something wrong? I've tried messing with various network settings on the Windows, no difference. (Windows 10 is a Bootcamp Mac Pro Trashcan, works fine with Dante 64 channels at 4ms...powerful enough...so I don't think it's a network issue, but maybe there's something specific to Audiogridder I'm unaware of). Thanks for any help...

**Comment by apohl79** [comment]
AG uses multiplies of the HW buffer size you have set in logic. You can change it in logics audio settings.

**Comment by dmuckala** [comment]
AG in Logic has 3 block sizes, depending on how you set Settings/Audio/Devices/Process Buffer Range.  It does not use I/O Buffer Size.  "Small" is 512 blocks, "Medium" is 1024 and "Large" is 2048.  I wish there were smaller options available, but there aren't.
